lib/category-name-converter.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
@scottnath Actually, 2 as the 1st param and no 2nd param means to use it within functions only. However, the case with no 2nd param is deprecated.